src/Atmos/Model/AtmosModel.jl

```diff
@@ -121,14 +123,19 @@ Where
 """
 @inline function flux_nondiffusive!(m::AtmosModel, flux::Grad, state::Vars,
                                     aux::Vars, t::Real)
-  ρinv = 1/state.ρ
+  ρ = state.ρ
+  ρinv = 1/ρ
   ρu = state.ρu
   u = ρinv * ρu
+  z  = altitude(m.orientation, aux)
 
   # advective terms
-  flux.ρ   = ρu
-  flux.ρu  = ρu .* u'
-  flux.ρe  = u * state.ρe
+  usub = subsidence_speed(m.subsidence, z)
```

Comment:
```suggestion
  usub = subsidence_velocity(m.subsidence, z)
```
("subsidence velocity" is the term we usually use)